hudi-client/hudi-client-common/src/test/java/org/apache/hudi/config/TestHoodieWriteConfig.java

```diff
@@ -43,6 +43,7 @@ public void testPropertyLoading() throws IOException {
     params.put(HoodieCompactionConfig.CLEANER_COMMITS_RETAINED_PROP.key(), "1");
     params.put(HoodieCompactionConfig.MAX_COMMITS_TO_KEEP_PROP.key(), "5");
     params.put(HoodieCompactionConfig.MIN_COMMITS_TO_KEEP_PROP.key(), "2");
+    params.put("hoodie.avro.schema.externalTransformation", "true");
```

Comment:
shouldn't we use EXTERNAL_RECORD_AND_SCHEMA_TRANSFORMATION.key() here?

Reply:
We can but that would have passed even without the fix. I thought to put what users would actually put in props file.